src/signals/incident-management/components/HiddenInput_b/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
zullen we die snapshot file weer weghalen?

Reply:
jazeker idd, dat is de file waar we het eerder over hadden heh!